### PR TITLE
(fix) O3-3743: Better success message when submitting a form.

### PR DESCRIPTION
**Pull request by IamMujuziMoses**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Currently when submitting a form successfully, the message “A new encounter was created“ is displayed. This PR changes the success message to a better success message that says “Form submitted successfully“.

## Screenshots

Screenshot 2024-08-08 at 9 46 42 PM

## Related Issue

https://issues.openmrs.org/browse/O3-3743

### CODE REVIEW

**Comment by IamMujuziMoses** [comment]
@ibacher could you please review this PR, your feedback will be appreciated.

File: translations/en.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
   "closesNotification": "Closes notification",
   "createdRecord": "Record created",
   "createdRecordDescription": "A new encounter was created",
```
Comment (ibacher) [inline translations/en.json:13]: Any ideas why these keys didn't get removed?

**Review by ibacher — APPROVED** [review]
Just one follow-up q, but this LGTM

File: translations/en.json (hunk 1 above)
Comment (IamMujuziMoses) [reply translations/en.json:13]: Thought they're being used somewhere else, is it okay if I remove them?!

Comment (ibacher) [reply translations/en.json]: AFAICT, they're only used in the spot we fixed... If they were used somewhere else, that's somewhere else we should fix.

**Review by samuelmale — APPROVED** [review]

**Pull request merged**